### DISCUSSION TITLE
SIGSEGV when testing in Jest environment

### DISCUSSION THREAD

**Discussion by deodad** [wevm/viem#856] [Question]
The issue, reproduced in [this repo](https://github.com/deodad/jest-viem-dynamic-import-sigsegv), is caused by using dynamic imports in Jest environments (see https://github.com/nodejs/node/issues/35889) and is not a problem specific to viem. 

I'm starting this discussion in case other viem users run into this issue. I'm also wondering if the maintainers have 1) encountered this and 2) have any suggestions to work around.

**Comment by tmm** [comment]
Haven't encountered this before and we use [Vitest](https://vitest.dev).
